src/raftstore/store/peer.rs

```diff
@@ -248,7 +248,7 @@ impl Peer {
             pending_cmds: Default::default(),
             peer_cache: store.peer_cache(),
             coprocessor_host: CoprocessorHost::new(),
-            size_diff_hint: 0,
+            size_diff_hint: cfg.region_check_size_diff,
```

Comment:
Yes. For example, a large region failed to split before shutdown. When restart, it won't check split until there is enough write operation.